environments/minikube/setup.go

```diff
@@ -25,20 +23,20 @@ func (minikube Minikube) runPostStartCommandsForMinikubeNoneDriver() {
 		fmt.Printf("Running %q\n", command)
 		output, err := execCommand(command)
 		if err != nil {
-			fmt.Printf("Running %q failed. Error: %+v\n", command, err)
-		} else {
-			fmt.Printf("Run %q successfully. Output: %s\n", command, output)
+			return err
```

Comment:
I think this should not return an error as many systems will not able to find `/home/.minikube` and `/home/.kube` directory because it is already moved to `root` directory. In this case, no error should be returned because these commands will not have any impact in starting the `minikube`

Reply:
Addressed